### PR TITLE
Fix/z position

### PR DESCRIPTION
**Pull request by lbebber**
#### What is the purpose of this pull request?
Use `react-portal` to add Popup and Sidebar on the top level of the DOM.

Before:
Screen Shot 2019-04-30 at 16 28 51

After:
Screen Shot 2019-04-30 at 16 28 56

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Popup.js [hunk 1]
```diff
@@ -9,29 +10,31 @@ import minicart from '../minicart.css'
...
-        >
-          <div className="shadow-3">
-            <div
-              className={`${
-                minicart.arrowUp
-              } absolute top-0 shadow-3 bg-base h1 w1 pa4 rotate-45`}
-            />
-            <div className="mt3 bg-base relative flex flex-column">
-              {children}
+        <Overlay>
```
Comment (lbebber) [inline react/components/Popup.js:21]: Basically the PR boils down to the inclusion of this component here...

File: react/components/Sidebar.js [hunk 2]
```diff
@@ -49,56 +50,57 @@ class Sidebar extends Component {
...
-
-        <Animation
-          className={`${
-            minicart.sidebar
-          } w-80 w-auto-ns h-100 fixed top-0 right-0 z-9999 bg-base shadow-2 flex flex-column`}
-          isActive={isOpen}
-          type="drawerLeft"
-        >
+    return (
+      <Overlay fullWidth>
```
Comment (lbebber) [inline react/components/Sidebar.js:54]: ...and this one

File: react/index.js [hunk 3]
```diff
@@ -248,47 +248,47 @@ class MiniCart extends Component {
...
-              {quantity > 0 && (
-                <span
-                  className={`${
-                    minicart.badge
-                    } c-on-emphasis absolute t-mini bg-emphasis br4 w1 h1 pa1 flex justify-center items-center lh-solid`}
-                >
-                  {quantity}
-                </span>
-              )}
+        <div className="flex flex-column">
```
Comment (lbebber) [inline react/index.js:251]: ...and this flex-column here.

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
```
Comment (lbebber) [inline CHANGELOG.md:11]: Please do select "Hide whitespace changes" on the Diff settings to get a better sense of what this PR does.

**Review by rerissondaniel — APPROVED** [review]

**Review by matheusps — APPROVED** [review]

**Pull request merged**